core/src/main/java/nl/nn/adapterframework/pipes/FixedResultPipe.java

```diff
@@ -261,33 +264,56 @@ public String getReturnString() {
 	public String getReplaceFrom() {
 		return replaceFrom;
 	}
+	/**
+	 * If set, every occurrence of this attribute's value is replaced by the value of <code>replaceTo</code>.
+	 * Not applied if returned message comes via attribute <code>filenameSessionKey</code> or if <code>lookupAtRuntime</code> is true.
```

Comment:
Als gespecificeerd wordt deze replacement nu in alle gevallen gedaan (a.g.v. aanpassing aan FixedResultPipe)

Reply:
Verwerkt.